config/hg19_ensembl.clean.yml

```diff
@@ -1,442 +1,443 @@
 ---
```

Comment:
likewise, mostly unchanged, only in fileProcessors, removal of non-existent 'files' field, and removal of nearest property